### ISSUE TITLE
Dockerhub

### ISSUE THREAD

**Issue by skky88** [optoisolated/MXWeather#5]
Hi, is possible have your container loaded in docker HUB?

**Comment by optoisolated** [comment]
Certainly; I will work on getting that submitted in the next week or two.

**Comment by skky88** [comment]
HI @optoisolated, any news about dockerHUB?

**Comment by zinz-code** [comment]
did you get around to it?
